soc/arm/gigadevice/gd32f4xx/Kconfig.series

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021, Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_GD32F4XX
+	bool "GigaDevice GD32F4XX series Cortex-M4F MCU"
+	select ARM
+	select CPU_HAS_ARM_MPU
+	select CPU_HAS_FPU
+	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_SYSTICK
```

Comment:
CPU_CORTEX_M_HAS_SYSTICK  and CPU_CORTEX_M_HAS_VTOR config are not required here, CPU_CORTEX_M4 select them.

Reply:
+1

Reply:
removed, thanks!